lib/rules/function-calc-no-unspaced-operator/index.js

```diff
@@ -166,6 +167,8 @@ const rule = (primary, _secondaryOptions, context) => {
 
 				if (firstNode.type !== 'word') return false;
 
+				if (!isStandardSyntaxValue(firstNode.value)) return false;
+
 				const operatorIndex = firstNode.value.search(OPERATOR_REGEX);
```

Comment:
[note] If the `isStandardSyntaxValue()` check is not performed, `firstNode.value` like `#{$foo-bar}` matches `/[+-]/` (`OPERATOR_REGEX`):

https://github.com/stylelint/stylelint/blob/e017583128863ee75634cb15e721779163b571e5/lib/rules/function-calc-no-unspaced-operator/index.js#L26